src/pages/saksbehandling/søknadsbehandling/vilkår/Vilkår.tsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 
-import { Person } from '~src/api/personApi';
+import { useOutletContext } from '~node_modules/react-router-dom';
```

Comment:
kan man ikke bare skriva `react-router-dom` istedet for å specificere fra node_modules?